api/Event.json

```diff
@@ -0,0 +1,1356 @@
+        }
+      },
+      "eventPhase": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Event/eventPhase",
+          "support": {
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "explicitOriginalTarget": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Event/explicitOriginalTarget",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "isTrusted": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Event/isTrusted",
+          "support": {
+            "chrome": {
+              "version_added": "46",
+              "notes": "Starting with Chrome 53 and Opera 40, untrusted events do not invoke the default action."
```

Comment:
Although you may be correct (especially given previous notes) the note that I am seeing doesn't mention Opera 40. @jpmedley ?

Reply:
It's correct. https://www.chromestatus.com/features/6461137440735232

On a side note, I always mentioned Chrome and Opera in the same note because it made the bottoms of the hand-built tables less messy. I don't do that in bcd JSON files because each browser must have its own note and they're tucked behind a control anyway.